resources/ext/ezbake.conf

```diff
@@ -2,8 +2,8 @@ ezbake: {
   pe: {}
   foss: {
     redhat: { dependencies: ["puppet >= 3.7.3", "puppet < 5.0.0"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] },
-    debian: { dependencies: ["puppet (>= 3.7.3-1puppetlabs1)", "puppet (<< 5.0.0-1puppetlabs1)"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] }
+              postinst: ["PATH=/opt/puppetlabs/bin:/usr/bin:$PATH puppetdb ssl-setup"] },
```

Comment:
Maybe the PATH belongs in the sub-command as well anyway? I mean if someone types that sub-command once it's installed, without the proper PATH it's going to break.